.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,35 @@
+name: Test
+
+on: [push]
```

Comment:
Since I haven't read through any of the CI/CD stuff for GitHub, can you explain the intended flow here? Does the Does this run on every push in a merge request?

Reply:
Currently runs on every push, the idea was to adjust and finalize this doc once the first tests are implemented.

To change to PRs, we simply the `on` paramter to `on: [pull_request]`

Docs are [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#about-workflow-events)

Reply:
Changed to `pull request` for now.